examples/get_iss_breast_data.py

```diff
@@ -12,16 +12,14 @@
 from starfish.types import Coordinates, Indices, Number
 from starfish.util.argparse import FsExistsType
 
-SHAPE = 1044, 1390
```

Comment:
This was here to avoid DRY.

Reply:
Had to take it out because I couldn't find a way to include a constant.